neuro_extras/main.py

```diff
@@ -240,7 +240,9 @@ async def _create_copier_container(
             image=neuro_api.RemoteImage.new_external_image("neuromation/neuro-extras"),
             resources=neuro_api.Resources(cpu=2.0, memory_mb=4096),
             volumes=volumes,
-            entrypoint=f"neuro-extras data cp {args}",
+            entrypoint=f"(gcloud auth activate-service-account "
```

Comment:
@anayden it could happen so that we do not have credentials for GCP, but the client wants to inject from AWS.
In such a case it will crash. We may think about calling one more `asyncio.create_subprocess_exec(command, *args)` [here](https://github.com/neuromation/neuro-extras/blob/master/neuro_extras/main.py#L417).
Sorry, didn't thought about that yesterday.

Reply:
Are you sure it'd crash here? My understanding is that since I use parenthesis and `||` (or) second command executes anyway, but I may be wrong of course

Reply:
Oh, you are right, it will continue executing the following commands (just tested it). 
Anyways, It might be confusing, but it's the balance between overcomplicated code and one false error msg.